src/app/GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1961,9 +1964,10 @@ internal enum Command
             OpenCommitsWithDifftool = 35,
             ToggleBetweenArtificialAndHeadCommits = 36,
             GoToChild = 37,
-            GoToParent = 38
+            GoToParent = 38,
 
             /* deprecated: RotateApplicationIcon = 14, */
+            ManageWorkTrees = 50,
```

Comment:
```suggestion
            ManageWorkTrees = 49,
```
and move up to section "// REPOSITORY menu", please.